server/index.js

```diff
@@ -32,13 +32,16 @@ app.get('/tiersIncluded', (req, res) => {
     res.json(tiersData);
   })
   .catch(err => {
-    res.status(500);
+    res.status(500).send(err);
   })
 });
 
 app.get('/tiersIncluded/:bundleId', (req, res) => {
-  const dataString = req.params;
-  return db.tiersRequest(dataString)
+  const dataObj = req.params;
+  if(dataObj.bundleId < 1 || dataObj.bundleId > 100 || typeof dataObj.bundleId !== 'number') {
```

Comment:
Do you want to handle a `bundleId` that is a floating point between 1 and 100? I've been using `parseInt()` as a precaution. See also lines 68 and 94.

Reply:
I'm not too worried about it, since it should be mostly system generated and if there's an invalid bundleId an error message will bounce up from the database. But something to keep in mind for other applications definitely.